packages/amplify_auth_plugin_interface/lib/src/Exceptions/AmplifyAuthException.dart

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_core/types/exception/AmplifyException.dart';
+
+/// Base Class for Auth Exceptions
+class AmplifyAuthException extends AmplifyException {
```

Comment:
Do we need to append `Amplify`? (Native doesn't)

Reply:
I was thinking this was the pattern we were establishing with `AmplifyAlreadyConfiguredException`... but, now I realize that Amplify was appended in this case because it was literally referring to the Amplify instance.  TLDR: No, we don't need to append it.